modular_skyrat/master_files/code/modules/client/preferences/middleware/limbs_and_markings.dm

```diff
@@ -2,7 +2,7 @@
 #define LIMBS_DEFAULT_NAME "None"
 
 /datum/preference_middleware/limbs_and_markings/post_set_preference(mob/user, preference, value)
-	preferences.character_preview_view.update_body()
+	// preferences.character_preview_view.update_body()
```

Comment:
? explain why removed or just remove fully

Reply:
I forgot to remove it fully, my bad, good catch.